src/ngAria/aria.js

```diff
@@ -83,7 +83,8 @@ function $AriaProvider() {
     ariaMultiline: true,
     ariaValue: true,
     tabindex: true,
-    bindKeypress: true
+    bindKeypress: true,
+    keypressNodeNames: ['DIV', 'LI']
```

Comment:
What if we just do away with the config for elements and limit dynamic binding of keypress to `div` and `li`? We could just wait and see if anyone actually needs it to be configurable.